change/@fluentui-react-avatar-e8c58b22-266b-4609-9169-1a26cdef500c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It looks like adding a mask changes the antialiasing around the user image, but it doesn't appear to be a consequential difference.

Otherwise, other changes look expected to me.

Reply:
I noticed that when setting a strong background like red, there's a gap between Badge and the ring around badge. It might just be the antialiasing messing up, but thought I would bring it up just in case.

![image](https://user-images.githubusercontent.com/5953191/233154376-e2c26342-b5ba-43c5-b7ab-701fa44bbe7c.png)

Reply:
If you're referring to the darker edge on the red badge in your screenshot, that's actually coming from Badge itself. The background color is bleeding through the edge of the badge's svg. It would need to be fixed on the badge itself.